packages/harmony/src/components/layout/Flex/Flex.stories.tsx

```diff
@@ -0,0 +1,86 @@
+import type { Meta, StoryObj } from '@storybook/react'
+
+import { Text } from '../../typography/Text'
+import { Box } from '../Box'
+
+import { Flex } from './Flex'
+
+const meta: Meta<typeof Flex> = {
+  title: 'Components/Layout/Flex',
+  component: Flex,
+  parameters: {
+    controls: { include: /^(alignItems|direction|gap|justifyContent|wrap)$/ }
+  },
+  args: {
+    gap: 'm'
+  },
+  argTypes: {
+    alignItems: {
+      control: { type: 'select' },
+      options: [
+        'baseline',
+        'flex-start',
+        'center',
+        'flex-end',
+        'space-around',
+        'space-between',
+        'space-evenly'
+      ]
+    },
+    justifyContent: {
+      control: { type: 'select' },
+      options: [
+        'flex-start',
+        'center',
+        'flex-end',
+        'space-around',
+        'space-between',
+        'space-evenly'
+      ]
+    },
+    gap: {
+      control: { type: 'select' },
+      options: ['xs', 's', 'm', 'l', 'xl', '2xl', '3xl', '4xl']
+    }
+  },
+  render: (props) => (
+    <Flex border='strong' borderRadius='m' p='l' mb='xl' {...props}>
+      <Box border='strong' borderRadius='s' p='s' ph='l'>
+        <Text variant='heading'>Box A</Text>
+      </Box>
+      <Box border='strong' borderRadius='s' p='s' ph='l'>
+        <Text variant='heading'>Box B</Text>
+      </Box>
+      <Box border='strong' borderRadius='s' p='s' ph='l'>
+        <Text variant='heading'>Box C</Text>
+      </Box>
+      <Box border='strong' borderRadius='s' p='s' ph='l'>
+        <Text variant='heading'>Box D</Text>
+      </Box>
+    </Flex>
+  )
+}
+
+export default meta
+
+type Story = StoryObj<typeof Flex>
+
+export const Default: Story = {}
```

Comment:
I'm tempted to say we should have more stories for different cases of flex, but there are plenty of resources available. Maybe we can link to some documentation?

It would be hard to be exhaustive, but perhaps we can sync with design on which types of layout are common on their side.

Reply:
Yea these were just some placeholder stories to get the idea down. I want to collab with design on which things we want to highlight in the stories